tools/test_utils.py

```diff
@@ -154,6 +154,12 @@ def run_ngtf_pytests_from_artifacts(artifacts_dir):
     # Next run the ngraph-tensorflow python tests
     command_executor(["pip", "install", "-U", "pytest"])
     command_executor(["pip", "install", "-U", "psutil"])
+
+    test_manifest_file = TestEnv.get_test_manifest_filename()
+    if not os.path.isabs(test_manifest_file):
+        test_manifest_file = os.path.join(test_dir, test_manifest_file)
+    os.environ['NGRAPH_TF_TEST_MANIFEST'] = test_manifest_file
+
```

Comment:
Not needed. If an absolute path is really needed, push this to wherever the env var is actually used.

Reply:
ok, removed isabs.